### ISSUE TITLE
Improve site design 

### ISSUE THREAD

**Issue by theshubhagrwl** [theshubhagrwl/policy-return-calculator#1]
Currently the site has a basic design. If you know CSS then you can try to improve this 😄 

Also the SUBMIT button is useless as all the calculations is done by the event listeners on the input fields.

**Comment by dollyjain51099** [comment]
I would like to work on this issue, please assign it to me!

**Comment by dollyjain51099** [comment]
Shall I remove the Submit button completely right?

**Comment by theshubhagrwl** [comment]
> Shall I remove the Submit button completely right?

Yeah, I was initially thinking to calculate on Submit but then I decided to use Event listeners to do the work kinda real time 😃

**Comment by dollyjain51099** [comment]
Cool

> > Shall I remove the Submit button completely right?
> 
> Yeah, I was initially thinking to calculate on Submit but then I decided to use Event listeners to do the work kinda real time 😃

Cool

**Comment by dollyjain51099** [comment]
> > Shall I remove the Submit button completely right?
> 
> Yeah, I was initially thinking to calculate on Submit but then I decided to use Event listeners to do the work kinda real time 😃

Cool!!

**Comment by dollyjain51099** [comment]
I have modified the background using CSS, kindly review my code and accept my PR in the Dev branch.
